src/Exception/MalformedRequestBodyException.php

```diff
@@ -10,11 +10,12 @@
 
 namespace Mezzio\Helper\Exception;
 
+use Exception;
 use InvalidArgumentException;
 
 class MalformedRequestBodyException extends InvalidArgumentException implements ExceptionInterface
 {
-    public function __construct($message, \Exception $previous = null)
+    public function __construct(string $message, ?Exception $previous = null)
```

Comment:
This is a BC break (change in signature due to adding the `string` type to `$message`).

Reply:
Reverted!